.gitignore

```diff
@@ -44,3 +44,4 @@ bin
 # Pack-ing artifacts
 packages/**/package
 **/*.tgz
+/examples/nextjs-app-router/
```

Comment:
I locally created an example to test against, mixed on if we need to include this as our examples are getting pretty big. But easy to add if we want it committed.